src/clientsettingsdlg.cpp

```diff
@@ -253,14 +253,14 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient*         pNCliP,
     edtNewClientLevel->setWhatsThis ( strNewClientLevel );
     edtNewClientLevel->setAccessibleName ( tr ( "New client level edit box" ) );
 
-    // custom central server address
-    QString strCentrServAddr = "<b>" + tr ( "Custom Central Server Address" ) + ":</b> " +
-        tr ( "Leave this blank unless you need to enter the address of a central "
+    // custom directory server address
+    QString strCentrServAddr = "<b>" + tr ( "Custom Directory Server Address" ) + ":</b> " +
```

Comment:
Shouldn't we also change variable names?

Reply:
I wasn't going to do variable names in this change - just the user visible strings, to match the documentation change.

Reply:
Ok. Makes sense.